scripts/jenkins/qat/plugin-deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Deploys current jenkins build test image 'intel-qat-plugin' in the cluster.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+sed -i "s#intel/intel-qat-plugin:devel#${REG}intel-qat-plugin:${TAG}#g" ./deployments/qat_plugin/qat_plugin.yaml
+sed -i "s#intel/crypto-perf:devel#${REG}crypto-perf:${TAG}#g" ./deployments/qat_dpdk_app/base/crypto-perf-dpdk-pod-requesting-qat.yaml
+kubectl create -f ./deployments/qat_plugin/qat_plugin_default_configmap.yaml
+kubectl create -f ./deployments/qat_plugin/qat_plugin.yaml
```

Comment:
Perhaps it would make sense to use an absolute path here calculated with e.g. 
```
REPO_ROOT=$(realpath $(dirname $0)/../../..)
...
kubectl create -f ${REPO_ROOT}/deployments/qat_plugin/qat_plugin_default_configmap.yaml
```

Reply:
@rojkov yeah agree :) Im sending a new commit with some more improvements.